tools/validators/ontology_validator/yamlformat/validator/entity_type_manager.py

```diff
@@ -440,3 +443,20 @@ def _MapTypenamesBySubset(self, types_to_include, field_to_typenames,
       typenames_by_subset[field_set].update(typegroup)
 
     return typenames_by_subset
+
+
+  def GetCompleteFieldSetsOI(self) -> dict():
+    """Returns a mapping of complete field sets to EntityType strings"""
```

Comment:
with complete you mean as the inherited fields included?

Reply:
Yes. This method is returning the mapping built by Analyze()